README.md

```diff
@@ -38,6 +38,15 @@
 
 ## Development
 
+### Running locally
+
+To try things quickly you can run `make run`. That will run `main.go` against
+a current kubectl context (i.e. `kubectl config current-context`).
+
+All flags that can be passed to the `main.go` can be also passed as environment
+variables prefixed with `CAPI_MIGRATION_`. E.g. `--provider=azure` can be
+set using `CAPI_MIGRATION_PROVIDER=azure make run`.
```

Comment:
This is how to solve the `make run-azure` problem. And other problems.

Reply:
Can you please provide an example for multi word flags such as --cloud-provider? does it require an env var named CAPI_MIGRATION_CLOUD_PROVIDER? or maybe CAPI_MIGRATION_CLOUD-PROVIDER? CAPI_MIGRATION_CLOUDPROVIDER? Just to avoid confusion, thanks